Week-3/4_Voting/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Write 2 Voting Contracts
+
+## One voting round
+1. Voting started and stopped by admin
+2. Candidates should register themselves before voting starts
+3. Each voter able to vote only once
```

Comment:
Add: Voters should be able to vote for registered candidated

Reply:
done